packages/vsx-registry/src/browser/style/index.css

```diff
@@ -49,6 +49,9 @@
 .theia-vsx-extension {
     display: flex;
     flex-direction: row;
+    padding: 0 0 0 calc(var(--theia-ui-padding)*4/3);;
+    line-height: 19px;
```

Comment:
I don't 19px is good, we should have some meaningful number. Look which variables we have.